packages/nextjs-kit/src/lib/sortChar.ts

```diff
@@ -0,0 +1,41 @@
+import { DataToSort, ASC, DESC } from '../types';
+
+/**
+ *
+ * @param {Array<Record<string, string | number>>} obj object to sort
+ * @param {string} key to sort object by
+ * @param {string} direction to sort object by
+ * @returns {Array<Record<string, string | number>>} the sorted object that was passed in
```

Comment:
Also here, we can omit the `Array<T>` type for `Type[]`
```suggestion
* @returns {Record<string, string | number>[]} An array of data sorted by the given key and direction
```